docs/modules/automation-testing/pages/scripting/create-image-injection-script.adoc

```diff
@@ -0,0 +1,98 @@
+= Create image injection script
+:navtitle: Create an image injection script
+
+Learn how to automate the process of image injection, so you can automate your test suite.
+
+== Before you start
+
+Enable image injection for your app with either of these options:
+
+* Either use the capability `'kobiton:instrumentenabled'` to xref:#appium_instrumentation_capability[instrument] your app.
+
+* Or add Kobiton image injection SDK into your Android or iOS apps. Follow xref:apps:image-injection-sdk/add-the-sdk-to-your-android-app.adoc[this guide] for andriod, and xref:apps:image-injection-sdk/add-the-sdk-to-your-ios-app.adoc[this guide] for iOS.
+
+[#appium_instrumentation_capability]
+== Appium capability (instrumentation only)
+
+Add the `'kobiton:instrumentenabled'` capability to your script’s capabilities list. The value of this capability must be `true` to instrument the app for image injection.
+
+
+[IMPORTANT]
+Only include this Appium capability for image injection via app instrumentation. *Do not* include it for apps with image injection SDK embedded.
+
+== Kobiton commands
+
+To automate the process of image injection, use commands provided by the Kobiton SDK.
+
+[options="header"]
+|=======================
+|Command | Description | Parameters
+| `kobiton:setImage` | Starts the image injection process by injecting a specified image into the camera view. | `data`: A base64 encoded string of the image you want to inject.
+| `kobiton:clearImage` |  This command stops the image injection process and reverts the camera view back to the live feed.
+ |
+|=======================
+
+=== Handle image file and injection
+
+Ensure the image file you intend to inject is stored locally and accessible by the test script.
+
+Pass the image file to the test script using `base64` encoding and include the command to inject it. Below is an example of an `injectImage` function demonstrating this process.
+
+.Example (javascript)

```

Comment:
javascript should be capitalized: JavaScript 
Please correct in all instances on this page

Reply:
All instances on of javascript updated to JavaScript